File: Python projects/Python Basics/while_loops/lab/max_number.py
from sys import maxsize
number = input()
max_number = - maxsize  # float('inf'), try it, works like a charm, judge likes it too

while number != 'Stop':
    if int(number) > max_number:
        max_number = int(number)
    number = input()

print(max_number)

File: JavaScript projects/JavaScript Basics/While Loops/lab/max_number.js
function maxNumber(input){
    let index = 0;
    let value = Number(input[0]);
    let maxNumber = Number(input[0]);

    while (value != "Stop"){
        if(maxNumber < value){
            maxNumber = Number(value);
        }
        index ++;
        value = input[index];
    }
    console.log(`${maxNumber}`)
}
maxNumber(["45", "-20", "7", "99", "Stop"]);